src/test/java/com/artipie/api/artifactory/GetPermissionSliceTest.java

```diff
@@ -129,25 +127,17 @@ void returnsUsersAndPermissionsList() {
                             .add(john, Json.createArrayBuilder().add("r").add("w").build())
                             .add(mark, Json.createArrayBuilder().add("m").build())
                             .build(),
-                        Collections.singletonList("**")
+                        "**/*"
                     )
                 ),
                 new RequestLine(RqMethod.GET, String.format("/api/security/permissions/%s", repo))
             )
         );
     }
 
-    private byte[] response(
-        final String repo,
-        final JsonObject users,
-        final Collection<String> patterns
-    ) {
-        final JsonArrayBuilder incpatterns = Json.createArrayBuilder();
-        for (final String pattern : patterns) {
-            incpatterns.add(pattern);
-        }
+    private byte[] response(final String repo, final JsonObject users, final String pattern) {
         return Json.createObjectBuilder()
-            .add("include-patterns", incpatterns)
+            .add("includesPattern", pattern)
             .add("repositories", Json.createArrayBuilder().add(repo).build())
```

Comment:
@olegmoz I'm not sure I like these changes as `include-patterns` is actually a list, would not it be better to revert changes here and add several patterns in one test method?

Reply:
@olenagerasimova that's not about test - it happens that original API has only one path in `GET` response, see https://www.jfrog.com/confluence/display/rtf/artifactory+rest+api#ArtifactoryRESTAPI-GetPermissionTargetDetails and `application/vnd.org.jfrog.artifactory.security.PermissionTarget+json` format description here: https://www.jfrog.com/confluence/display/JFROG/Security+Configuration+JSON

So original implementation was wrong. I noticed it only updating the docs

Reply:
@olegmoz oh, I see, thanks